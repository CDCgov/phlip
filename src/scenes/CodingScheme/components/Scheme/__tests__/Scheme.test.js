import React from 'react'
import { shallow } from 'enzyme'
import { Scheme } from '../index'

const props = {
  questions: [],
  handleQuestionTreeChange: () => {},
  handleHoverOnQuestion: () => {},
  enableHover: () => {},
  disableHover: () => {}
}

describe('CodingScheme scene -- Scheme component', () => {
  test('should render correctly', () => {
    expect(shallow(<Scheme {...props} />)).toMatchSnapshot()
  })
})
