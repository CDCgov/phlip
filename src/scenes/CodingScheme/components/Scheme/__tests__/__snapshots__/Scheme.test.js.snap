// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CodingScheme scene -- Scheme component should render correctly 1`] = `
<DragDropContext(ReactSortableTree)
  canDrop={[Function]}
  generateNodeProps={[Function]}
  isVirtualized={true}
  onChange={[Function]}
  style={
    Object {
      "flex": "1 0 50%",
    }
  }
  theme={
    Object {
      "nodeContentRenderer": [Function],
      "reactVirtualizedListProps": Object {
        "overscanRowCount": 10,
      },
      "rowHeight": 75,
      "scaffoldBlockPxWidth": 100,
      "slideRegionSize": 50,
      "treeNodeRenderer": [Function],
    }
  }
  treeData={Array []}
/>
`;
